workflow/scripts/sbs_process/eval.py

```diff
@@ -0,0 +1,91 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from lib.sbs_process.eval import (
+    load_and_concatenate_hdfs,
+    plot_mapping_vs_threshold,
+    plot_mapping_vs_threshold,
+    plot_read_mapping_heatmap,
+    plot_cell_mapping_heatmap,
+    plot_reads_per_cell_histogram,
+    plot_gene_symbol_histogram,
+)
+
+# Read barcodes
+df_design = pd.read_csv(snakemake.params.df_design_path, sep="\t")
+df_pool = df_design.query("dialout==[0,1]").drop_duplicates("sgRNA")
+df_pool["prefix"] = df_pool.apply(lambda x: x.sgRNA[: x.prefix_length], axis=1)
+barcodes = df_pool["prefix"]
+
+# Concatenate files
+reads = load_and_concatenate_hdfs(snakemake.input.read_files)
+cells = load_and_concatenate_hdfs(snakemake.input.cell_files)
+minimal_phenotype_info = load_and_concatenate_hdfs(
+    snakemake.input.minimal_phenotype_info_files
+)
+
+plot_mapping_vs_threshold(reads, barcodes, "peak")
+plt.gcf().savefig(snakemake.output[0])
+plt.close()
+
+plot_mapping_vs_threshold(reads, barcodes, "Q_min")
+plt.gcf().savefig(snakemake.output[1])
+plt.close()
+
+plot_read_mapping_heatmap(reads, barcodes, shape="6W_sbs")
+plt.gcf().savefig(snakemake.output[2])
+plt.close()
+
+df_summary_one, _ = plot_cell_mapping_heatmap(
+    cells,
+    minimal_phenotype_info,
+    barcodes,
+    mapping_to="one",
+    mapping_strategy="gene_symbols",
+    shape="6W_sbs",
```

Comment:
Please note that in the phenotyping step, the shape may incorrectly be 6W_sbs. I will remember this and bring it up later as well. Good here.